src/game/client/worldheightmap.h

```diff
@@ -135,17 +197,49 @@ class WorldHeightMap : public RefCountClass, public WorldHeightMapInterfaceClass
     }
 
     unsigned char *Get_Pointer_To_Tile_Data(int x, int y, int width);
+    void Get_UV_For_Ndx(int tile_ndx, float *min_u, float *max_u, float *min_v, float *max_v, bool full_tile);
+    bool Get_UV_For_Tile_Index(int ndx, short tile_ndx, float *const v, float *const u, bool full_tile);
+    int Get_Texture_Class_From_Ndx(int tile_ndx);
+    unsigned char *Get_RGB_Alpha_Data_For_Width(int width, TBlendTileInfo *info);
+    bool Get_Raw_Tile_Data(short tile_ndx, int width, unsigned char *buffer, int size);
+
+    void Set_Cell_Cliff_Flag_From_Heights(int x_index, int y_index);
+    void Set_Cliff_State(int x_index, int y_index, bool state);
+
+    void Clear_Seismic_Update_Flags();
+    int Update_Tile_Texture_Positions(int *edge_height);
+    void Fill_Seismic_Z_Velocities(float value);
+    void Init_Cliff_Flags_From_Heights();
+    void Setup_Alpha_Tiles();
+
+    bool Parse_Blend_Tile_Data(DataChunkInput &file, DataChunkInfo *info, void *user_data);
+    bool Parse_Height_Map_Data(DataChunkInput &file, DataChunkInfo *info, void *user_data);
+    bool Parse_Object_Data(DataChunkInput &file, DataChunkInfo *info, void *user_data, bool read_dict);
+    bool Parse_Size_Only(DataChunkInput &file, DataChunkInfo *info, void *user_data);
+
+    static bool Parse_Blend_Tile_Data_Chunk(DataChunkInput &file, DataChunkInfo *info, void *user_data);
+    static bool Parse_Height_Map_Data_Chunk(DataChunkInput &file, DataChunkInfo *info, void *user_data);
+    static bool Parse_Lighting_Data_Chunk(DataChunkInput &file, DataChunkInfo *info, void *user_data);
+    static bool Parse_Object_Data_Chunk(DataChunkInput &file, DataChunkInfo *info, void *user_data);
+    static bool Parse_Objects_Data_Chunk(DataChunkInput &file, DataChunkInfo *info, void *user_data);
+    static bool Parse_Size_Only_In_Chunk(DataChunkInput &file, DataChunkInfo *info, void *user_data);
+    static bool Parse_World_Dict_Data_Chunk(DataChunkInput &file, DataChunkInfo *info, void *user_data);
+
+    static int Count_Tiles(InputStream *str, bool *b);
+    static void Free_List_Of_Map_Objects();
+    static bool Read_Tiles(InputStream *str, TileData **tiles, int num_rows);
+    void Read_Tex_Class(TXTextureClass *tex_class, TileData **tile_data);
 
     int m_width;
     int m_height;
     int m_borderSize;
     std::vector<ICoord2D> m_boundaries;
     int m_dataSize;
     unsigned char *m_data;
-    unsigned char *m_siesmicUpdateFlags;
+    unsigned char *m_seismicUpdateFlags;
     int m_siesmicUpdateWidth;
     float *m_siesmicZVelocity;

```

Comment:
m_seismicZVelocities apparently, WorldHeightMap::ParseHeightMapData prints it

Reply:
fixed.